sdk/confidentialledger/Azure.Security.CodeTransparency/tests/CodeTransparencyClientLiveTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Security.CodeTransparency.Tests
+{
+    public class CodeTransparencyClientLiveTests: ClientTestBase
```

Comment:
This is the only file for Live tests that I see, and it looks a bit bare.   We should have live test coverage for all service operations supported by the library.

Reply:
we cannot have those as the resources cannot be created using ARM and the existing infrastructure